src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -87,23 +87,24 @@ def calcEvtsPerJobLumi(ePerJob, ePerLumi, tPerEvent):
         Given EventsPerJob, EventsPerLumi and TimePerEvent information,
         calculates the final values for EventsPerJob and EventsPerLumi.
         
-        If user provided both values, then we don't do anything.
+        Final result will always be an EventsPerJob multiple of EventsPerLumi,
+        no matter whether EventsPerJob was provided or not.
         :param ePerJob: events per job
         :param ePerLumi: events per lumi
         :param tPerEvent: time per event
         """
-        if ePerLumi is None and ePerJob is None:
-            ePerJob = int((8.0 * 3600.0) / tPerEvent)
+        if ePerLumi is None:
+            if ePerJob is None:
+                ePerJob = int((8.0 * 3600.0) / tPerEvent)
             ePerLumi = ePerJob
-        elif ePerJob is None:
-            ePerJob = int((8.0 * 3600.0) / tPerEvent)
+        else:
+            if ePerJob is None:
+                ePerJob = int((8.0 * 3600.0) / tPerEvent)
             # then make EventsPerJob multiple of EventsPerLumi and still closer to 8h jobs
```

Comment:
Sorry Alan for the late review. 
It might be clearer 

```python
# if eventPerJob is not set set to 8 hours
if ePerJob is None:
    ePerJob = int((8.0 * 3600.0) / tPerEvent)
if ePerLumi is None:
    ePerLumi = ePerJob
else:
    # then make EventsPerJob multiple of EventsPerLumi and still closer to 8
    multiplier = int(round(ePerJob / ePerLumi))
     # make sure not to have 0 EventsPerJob
     multiplier = max(multiplier, 1)
      ePerJob = ePerLumi * multiplier
```

Reply:
Thanks, Seangchan. Your code is more readable indeed. Let me update this patch.